jupyterlab/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jupyter/tensorflow-notebook:latest
```

Comment:
See the codacy inspection for this line. This should be pinned to a specific version because:
1. Later versions may introduce breaking changes that we don't want prorogating into production
2. The spawner docs encourage keeping the jupyter/jupyterhub versions consistent across containers

Reply:
Done